### PR TITLE
The long awaited big Advanced Reports cleanup

### PR DESCRIPTION
**Pull request by jgeskens**
During the last weeks, I worked on polishing the Advanced Reports project. It has no backwards incompatible changes, but mostly cleanups and serious increase of examples. The test project also got a lot of love. Enjoy the ride!

This branch runs on http://backoffice.oemfoeland.com. You can login there using admin/admin.

### CODE REVIEW

**Comment by jonathanslenders** [comment]
I like the commits! :)
No, serious comments for now actually, but I don't have a lot of knowledge about advanced-reports.

File: advreport_test_project/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-Django
-django-widget-tweaks
+Django==1.7.2
+django-widget-tweaks==1.3
+Pygments==2.0.1
```
Comment (borzecki) [inline advreport_test_project/requirements.txt]: I would add `django-advanced-reports` to the requirements :)

**Comment by borzecki** [comment]
Made a test drive of this version and don't see any potential bugs. I really like the way your making your commits, really made the whole easier to review :+1: Examples make it easy to quickly plug and play in any project. I've few ideas for improvements but that would definitely screw up backward compatibility, but that's something to discuss later if you're interested.

Beside things I've already mentioned it's great and good to go :shipit:

**Comment by jgeskens** [comment]
Ok, I've waited long enough now for feedback. Thanks @jonathanslenders and @borzecki! Let's merge this baby.

File: advreport_test_project/requirements.txt (hunk 1 above)
Comment (jgeskens) [inline advreport_test_project/requirements.txt]: Yeah, but it's only for the test project. I'm not sure if local directories can be put here as well?
On the CI deployment I just do an extra setup.py develop on the advanced reports.

**Pull request merged**